### PR TITLE
Update button-card.js to work in HA 0.88

### PR DESCRIPTION
**Pull request by ammmze**
Have not tested it in previous versions to see what happens in those versions 🤷‍♂️

This is for issue #57

The result looks something like this...

home assistant 2019-02-20 21-16-26

### CODE REVIEW

File: button-card.js [hunk 1]
```diff
@@ -107,8 +107,12 @@ class ButtonCard extends LitElement {
     }
     paper-button {
       display: flex;
+      flex-direction: column;
       margin: auto;
       text-align: center;
+      align-items: center;
+      text-transform: uppercase;
```
Comment (ammmze) [inline button-card.js]: I never had an instance displaying text. I just assumed the previous `paper-button` did it uppercase based on the screenshots in the `button-card` README. But if that is not the case, we can pull this off.

**Comment by arsaboo** [comment]
With this code, all the colors are lost for me. Everything is black and white and the buttons don't reflect the state.

**Comment by DavidFW1960** [comment]
I tried thas and I also tried merging your changes with @iantrich changes in his pull request (to use the mwc button) and still used your extra <div> additions...

What is working best for me is the css code I posted in the issue I created on kuuji's github along with the first change to the new element.

Using that all my colours are right but it's still using paper-button of course. I believe the good stuff from kuuji's button is going into core anyway so it's a waiting game. It's a pity Kuuji is no longer maintaining this button.

**Comment by iantrich** [comment]
He’s agreed to test and merge PRs. It’s effectively a community card now, so solutions need to come from the community

**Comment by DavidFW1960** [comment]
ok... @ammmze... still using paper-button though... not mwc-button...

**Comment by DavidFW1960** [comment]
So hopefully I can go back to my normal styling pre-0.88 and it should all be nice again? Great job.

**Comment by bouwew** [comment]
When I use the latest code from here: https://github.com/ammmze/button-card/raw/24e68e30cfb51890f2ee0f52fc05a3b4e59d8d50/button-card.js
I get this error: http://192.168.xxx.yyy:8123/local/button-card.js:1:39 Uncaught TypeError: Cannot convert undefined or null to object

**Comment by DavidFW1960** [comment]
Did you do a hard refresh and clear cache? In chrome, F12 and rightclick refresh and select the hard reload and clear cache. The new one is working for me... the button is still slightly smaller than the normal button-card and previous custom-button-card but it works...

**Comment by bouwew** [comment]
Home Assistant throws this error, not Chrome.

**Comment by tbkon3** [comment]
Same error for me: https://XXXX.duckdns.org:XXXX/local/custom_ui/button-card/button-card.js:1:39 Uncaught TypeError: Cannot convert undefined or null to object

Position of my button-card is correct (www/custom_ui/button-card/).

**Pull request closed without merging**